### DISCUSSION TITLE
Adding a note to the README.md that Quartz is practically a dead project

### DISCUSSION THREAD

**Discussion by pbartoszek** [kagkarlsson/db-scheduler#508] [General]
After looking at Quartz commit history but also people asking about future of Quartz I got impression that Qurtaz is not longer maintained. I think it would be even better to make it very clear in README.md file that db-scheduler is the only truly free choice for job scheduling in java!

**Comment by kagkarlsson** [comment]
Yeah, good point, I think Quartz is less of an alternative now. I think the FAQ is something like 8 years old 😄
